### PR TITLE
add more unit test for ComputeNodePersistService (#14835)

### LINKED ISSUE #14835: Add test case for ComputeNodePersistService

### PR DESCRIPTION
**Pull request by art-yang**
Fixes #14835

Changes proposed in this pull request:
- add more unit test for ComputeNodePersistService

### CODE REVIEW

File: shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/metadata/persist/service/ComputeNodePersistServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import java.util.Collections;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.isA;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ComputeNodePersistServiceTest {
```
Comment (menghaoranss) [inline shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/metadata/persist/service/ComputeNodePersistServiceTest.java:41]: please add `final`

File: shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/metadata/persist/service/ComputeNodePersistServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        assertThat(actual, isA(Collection.class));
+    }
+
+    @Test
+    public void assertLoadComputeNodeInstance() {
+        InstanceDefinition instanceDefinition = new InstanceDefinition(InstanceType.PROXY, 3307);
+        ComputeNodeInstance actual = new ComputeNodePersistService(repository).loadComputeNodeInstance(instanceDefinition);
+        assertThat(actual.getInstanceDefinition(), is(instanceDefinition));
+    }
+
```
Comment (menghaoranss) [inline shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/metadata/persist/service/ComputeNodePersistServiceTest.java:108]: remove unnecessary empty line.

**Review by menghaoranss — CHANGES_REQUESTED** [review]
thanks for your contribution, I just leave some suggestions about code style, looking forward to your correction

File: shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/metadata/persist/service/ComputeNodePersistServiceTest.java (hunk 1 above)
Comment (art-yang) [reply shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/metadata/persist/service/ComputeNodePersistServiceTest.java]: The final keyword has been added

File: shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/metadata/persist/service/ComputeNodePersistServiceTest.java (hunk 2 above)
Comment (art-yang) [reply shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/metadata/persist/service/ComputeNodePersistServiceTest.java]: Unnecessary empty line have been removed

**Review by terrymanu — APPROVED** [review]

**Review by menghaoranss — APPROVED** [review]

**Pull request merged**